### ISSUE TITLE
Improve docs on the behaviour of subscription creation with "unclean" event types

### ISSUE THREAD

**Issue by FriedrichWilken** [kyma-project/website#816]
**Description**

For the creation of Subscriptions non-alphanumeric characters (further called nacs) are not allowed for the event type value. If a Subscription gets created with nacs in the event type value, the nacs will get removed automatically. At the same time, the specs that contain the nacs are not getting touched.

So this:
```yaml
spec:
    filter:
      filters:
      - eventSource:
          property: source
          type: exact
          value: ""
        eventType:
          property: type
          type: exact
          value: sap.kyma.custom.mp-frog-ec-202.customer.created.v1
    sink: http://test-ec-lambda.frog3.svc.cluster.local 
```
will create a subscription with the event type `sap.kyma.custom.mpfrogec202.customer.created.v1` while the specs will continue to display this:
```yaml
...
value: sap.kyma.custom.mp-frog-ec-202.customer.created.v1
...
```

This causes ongoing confusion.

We already created a [PR](https://github.com/kyma-project/kyma/pull/12488) that will also show the`cleanEventTypes` in addition to the user created specs.

We should also improve the [docs](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/evnt-01-subscription) to point out that:

- nac are not allowed for eventTypeValue
- if nacs are still used for creating a Subscription, they will get automatically removed
- even when the event typeValue got cleaned up, the user input stays untouched
- only after the aforementioned PR got merged: document the new `cleanEventTypes` field's behaviour. 

The behaviour is partially explained [here](https://kyma-project.io/docs/kyma/latest/05-technical-reference/evnt-01-event-names). We should reference the section in the subscriptions doc.

**Area**

[docs/ reference/ cr/ subscription](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/evnt-01-subscription)

**Reasons**

Ongoing confusion

**Assignees**

To be discussed in the team

**Attachments**

-

**Comment by alexandra-simeonova** [comment]
The PR for this issue is here: https://github.com/kyma-project/kyma/pull/13112
@FriedrichWilken In the future, please create documentation issues in the https://github.com/kyma-project/kyma repository instead of `website`, as this is where the source code for the docs resides. Thank you!

**Comment by alexandra-simeonova** [comment]
The PR is merged kyma-project/kyma#13112
